scripts/shared/lib/utils

```diff
@@ -120,12 +116,26 @@ function declare_cidrs() {
     # shellcheck disable=SC2034 # these variables are used elsewhere
     declare -gA cluster_CIDRs service_CIDRs global_CIDRs
 
-    for i in {1..3}; do
-        add_cluster_cidrs $i
+    i=1
+    for cluster in ${clusters[*]}; do
+        add_cluster_cidrs "$i" "$cluster"
+        i=$(("$i"+1))
     done
 }
 
 function declare_kubeconfig() {
     export KUBECONFIG
-    KUBECONFIG=$(echo "${KUBECONFIGS_DIR}"/kind-config-cluster{1..3} | sed 's/ /:/g')
+    KUBECONFIG=$(find "${KUBECONFIGS_DIR}" -type f | tr '\n' ':')
+}
+
+function print_clusters_message() {
+    cat << EOM
+Your virtual cluster(s) are deployed and working properly and can be accessed with:
+
+export KUBECONFIG=\$(find \$(git rev-parse --show-toplevel)/output/kubeconfigs/ -type f | tr '\n' ':')
+
+$ kubectl config use-context cluster1 # or cluster2, cluster3..
+
+To clean everthing up, just run: make cleanup
+    EOM
```

Comment:
```suggestion
EOM
```

This should fix the CI (I know it's confusing. but that's linux antics for you :))

Reply:
I merged it right away so we can check CI.